pygeoapi/process/manager/base.py

```diff
@@ -211,7 +220,11 @@ def _execute_handler_sync(self, p: BaseProcessor, job_id: str,
         :param p: `pygeoapi.process` object
         :param job_id: job identifier
         :param data_dict: `dict` of data parameters
-
+        :param out_dict: `dict` optionally specify the subset of required
+            outputs - defaults to all outputs.
+            The value of any key may be an object and include the property
+            `transmissionMode` - defauts to `value`.
+            
         :returns: tuple of MIME type, response payload and status
         """
 
```

Comment:
Don't we also need to forward the `out_dict` parameter to the `execute()` call below in the line containing this?
```jfmt, outputs = p.execute(data_dict)```

Btw what do you think about calling the parameter `outputs` everywhere instead of `out_dict` here? It's already called `outputs` in the API and also in the `BaseProcessor`